frb/dm_kde/transient_kdes.py

```diff
@@ -0,0 +1,133 @@
+""" Definitions relating to the observed DM distributions of pulsars and FRBs.
+Standard KDE is used for pulsars and varying assymetric KDE for FRBs.
+To use this code, please download the asymmetric_kde package from
+https://github.com/tillahoffmann/asymmetric_kde [Hoffman and Jones, 2015] """
+
+from asymmetric_kde import ProperGammaEstimator
+import numpy as np
+import pandas as pd
+from scipy.signal import argrelextrema
+from pdf_fns import make_kde_funtion
```

Comment:
is this in a standard anaconda install?  if not, 
we need to add it to the optional install notes, i.e.
https://github.com/FRBs/FRB/blob/master/docs/installing.rst

Reply:
Same with `asymmetric_kde`

Reply:
pdf_fns is just a file in the repo

Reply:
Will add to docs for asymmetric kde, thanks!

Reply:
Done :)